source/tokens/contracts/OMGToken.sol

```diff
@@ -7,6 +7,7 @@
 pragma solidity 0.5.12;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
```

Comment:
if youre now importing Ownable, why not import the ERC20Basic, BasicToken, etc?

Reply:
ERC20Basic and BasicToken do not exist anymore in the modern openzeppelin version (2.4).  Most of OMGToken is pulled from their etherscan upload. Ownable is imported because when i copied just Ownable into this file, the compiler would warn that another Ownable contract existed bc we import Ownable from openzeppelin in other contracts in the tokens/airswap-protocotol.